lib/_http_server.js

```diff
@@ -645,7 +636,7 @@ function onParserTimeout(server, socket) {
     socket.destroy();
 }
 
-const noop = FunctionPrototype;
+const noop = function() {};
```

Comment:
I have some good suspicion that using a noop function defined elsewhere is not optimized away. No hard data to prove it.

Reply:
We've seen that before. Not sure what causes it but reused noops definitely cause issues.

Reply:
All noop functions have been replaced by the `FunctionPrototype` in most of the files I checked  :/.